snippets/resolver/named.conf

```
@@ -1,28 +1,48 @@
 options {
-  directory "/var/cache/bind";
-  listen-on { any; };
-  listen-on-v6 { any; };
-  pid-file "/run/named/named.pid";
+directory "/var/cache/bind";
+allow-recursion {
+192.168.0.0/24;
+127.0.0.1/32;
 };
-
-zone "2tm1-admin-sys2.ephec.austreelis.net" IN {
-  type master;
-  file "/etc/bind/2tm1-admin-sys2.ephec.austreelis.net.zone";
+// Configure the IPs to listen on here.
+listen-on port 53 {any;};
+listen-on-v6 { none; };
+pid-file "/var/run/named/named.pid";
+allow-transfer { none; };
+};
+zone "." IN {
+type hint;
+file "/usr/share/dns/root.hints";
+};
+zone "localhost" IN {
+type master;
+file "db.local";
+allow-update { none; };
+notify no;
+};
```

Fix indentation:

```suggestion
zone "localhost" IN {
  type master;
  file "db.local";
  allow-update { none; };
  notify no;
};

```

Also, that db file isn't included in the docker image. I'm also not sure we need such a zone for a resolver (even a local one).

---

The file comes with the bind package. I checked and it is present.  I think the role of such is not is to link the hostname "localhost" to the loopback addresses "127.0.0.1" and "::1"